tasks/move-granules/index.js

```diff
@@ -25,13 +25,16 @@ const {
 } = require('@cumulus/cmrjs');
 
 const BucketsConfig = require('@cumulus/common/BucketsConfig');
-
 const { urlPathTemplate } = require('@cumulus/ingest/url-path-template');
 const { isFileExtensionMatched } = require('@cumulus/message/utils');
 const log = require('@cumulus/common/log');
+const { constructCollectionId } = require('@cumulus/message/Collections');
 
 const MB = 1024 * 1024;
-
+/**
+* @typedef { import('@cumulus/types').ApiGranuleRecord } ApiGranuleRecord
```

Comment:
I don't think this type is right - an ApiGranuleRecord, as it's not for sure going to have some of the required keys.